src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/xlf/CompilerExtensionsResources.cs.xlf

```diff
@@ -17,11 +17,6 @@
         <target state="translated">Element se stejným klíčem, ale odlišnou hodnotou už existuje.</target>
         <note />
       </trans-unit>
-      <trans-unit id="ArgumentException_OtherNotArrayOfCorrectLength">
-        <source>Object is not a array with the same number of elements as the array to compare it to.</source>
-        <target state="translated">Objekt není pole se stejným počtem elementů jako pole, se kterým se má porovnat.</target>
-        <note />
-      </trans-unit>
```

Comment:
An aside, but in dotnet/project-system we mark `.xlf` files as generated so that they're collapsed by default in GitHub diffs.

https://github.com/dotnet/project-system/commit/3ba729520b8540f4496621b25698b33d36e3cabe

Reply:
I mentioned this a long time ago, but it was not adopted for this repository at the time.